torch/onnx/_internal/fx/passes/type_promotion.py

```diff
@@ -70,14 +70,19 @@ class TypePromotionSnapshot:
 
 
 @_beartype.beartype
-def _fake_tensor_from_node_val(node: torch.fx.Node) -> fake_tensor.FakeTensor:
+def _fake_tensor_from_node_val(
+    node: torch.fx.Node,
+) -> Union[fake_tensor.FakeTensor, int, float, bool]:
     """Syntactic sugar for retrieving fake tensor from node.meta['val']."""
     val = node.meta.get("val", None)
-    if not isinstance(val, fake_tensor.FakeTensor):
+    if isinstance(val, fake_tensor.FakeTensor):
+        return val
+    elif isinstance(val, (torch.SymInt, torch.SymFloat, torch.SymBool)):
+        return val.node.hint
```

Comment:
Could this return None? I don't find `node.hint` usage under torch.fx.Node. Is it in somewhere else? I was using `hint_int`, and would like to know more about this.